src/learning/flare/flr-dca-cooldown.rain

```diff
@@ -55,13 +54,11 @@ charts:
       - label: IO ratio
         value: 0.6
 ---
-#flare-sub-parser !The flare subparser address.
-
 #per-trade-amount !The amount that we'll buy or sell per trade.
 #cooldown-time !The minimum amount of seconds that must elapse between trades.
 
 #calculate-io
-using-words-from flare-sub-parser
+using-words-from 0xAfD94467d2eC43D9aD39f835BA758b61b2f41A0E 0x50fB1Be59721E7A2a3434dDd726d53780bfDe344
```

Comment:
why did you remove this from being a binding?

Reply:
Because the strats are specifically for flare anyway and I think this way it's a clearer example of how subparsers work